scripts/photon-containers.sh

```diff
@@ -47,4 +47,47 @@ ANTREA_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["antrea"].gitRepoTag')
-sed -i '/image:.*/i \        imagePullPolicy: IfNotPresent' /root/download/antrea.yml
\ No newline at end of file
+sed -i '/image:.*/i \        imagePullPolicy: IfNotPresent' /root/download/antrea.yml
+
+echo '> Downloading Knative...'
+KNATIVE_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["knative"].gitRepoTag')
+curl -L https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving-crds.yaml -o /root/download/serving-crds.yaml
+curl -L https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving-core.yaml -o /root/download/serving-core.yaml
+
+curl -L https://github.com/knative/eventing/releases/download/${KNATIVE_VERSION}/eventing-crds.yaml -o /root/download/eventing-crds.yaml
+curl -L https://github.com/knative/eventing/releases/download/${KNATIVE_VERSION}/eventing-core.yaml -o /root/download/eventing-core.yaml
+
+echo '> Downloading RabbitMQ...'
+RABBITMQ_OPERATOR_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["rabbitmq-operator"].gitRepoTag')
+curl -L https://github.com/rabbitmq/cluster-operator/releases/download/${RABBITMQ_OPERATOR_VERSION}/cluster-operator.yml -o /root/download/cluster-operator.yml
+sed -i '/image: rabbitmqoperator.*/i \        imagePullPolicy: IfNotPresent' /root/download/cluster-operator.yml
+
+RABBITMQ_BROKER_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["rabbitmq-broker"].gitRepoTag')
+curl -L https://github.com/knative-sandbox/eventing-rabbitmq/releases/download/${RABBITMQ_BROKER_VERSION}/rabbitmq-broker.yaml -o /root/download/rabbitmq-broker.yaml
+
+echo '> Downloading Knative Contour...'
+curl -L https://github.com/knative/net-contour/releases/download/${KNATIVE_VERSION}/contour.yaml -o /root/download/knative-contour.yaml
+sed -i '1902i\      dnsPolicy: ClusterFirstWithHostNet\n      hostNetwork: true' /root/download/knative-contour.yaml
+sed -i 's/type: LoadBalancer/type: NodePort/g' /root/download/knative-contour.yaml
+sed -i 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' /root/download/knative-contour.yaml
+cat > /root/download/contour-delegation.yaml << EOF
+apiVersion: projectcontour.io/v1
+kind: TLSCertificateDelegation
+metadata:
+  name: default-delegation
+  namespace: contour-external
+spec:
+  delegations:
+    - secretName: default-cert
+      targetNamespaces:
+      - "*"
+EOF
+curl -L https://github.com/knative/net-contour/releases/download/${KNATIVE_VERSION}/net-contour.yaml -o /root/download/net-contour.yaml
+
+echo '> Downlaoding Local Path Provisioner...'
+LOCAL_PROVISIONER_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["csi"].gitRepoTag')
+LOCAL_STOARGE_VOLUME_PATH="/data/local-path-provisioner"
+curl -L https://raw.githubusercontent.com/rancher/local-path-provisioner/${LOCAL_PROVISIONER_VERSION}/deploy/local-path-storage.yaml -o /root/download/local-path-storage.yaml
+sed -i "s#/opt/local-path-provisioner#${LOCAL_STOARGE_VOLUME_PATH}#g" /root/download/local-path-storage.yaml
+sed -i 's/busybox/busybox:latest/g' /root/download/local-path-storage.yaml
```

Comment:
if you use `latest` here, image will always be pulled (irrespective of next line), no?

Reply:
Without adding `:latest` to the manifest, I found in the non-internet connected scenario, that it would throw image pull error. Once appended, when the local provisioner is setup (suspect busybox is used for testing purposes), then it was successful

Reply:
Re-Verified, works as expected in non-internet connected environment